scripts/data/supersearch_fields.json

```diff
@@ -338,9 +336,7 @@
         "is_returned": false,
         "query_type": "enum",
         "permissions_needed": [],
-        "storage_mapping": {
-            "type": "string"
-        },
+        "storage_mapping": null,
```

Comment:
That this field will not have a storage mapping in Elasticsearch. Basically the impact is very low, we will just rely on ES's default mechanism. I'm doing this here because that field is not exposed or returned in any way, it's an old field that we don't use anymore.